goss/goss.yaml

```diff
@@ -1,3 +1,11 @@
+{{ $distro := .Vars.OS -}}
+{{ if contains "ubuntu" $distro -}}
+  {{ $distro = "ubuntu" -}}
+{{ end -}}
```

Comment:
Unfortuantly the ubuntu `distributions` / offers in azure are kinda cray cray. `0001-com-ubuntu-server-focal-20_04-lts-gen2` and `ubuntuserver-18_04-lts-gen2` so normalizing them here like this to get the goss import to work correctly.